Content.Shared/Atmos/Atmospherics.cs

```diff
@@ -274,6 +274,124 @@ static Atmospherics()
         public const float MaxTransferRate = 200;
 
         #endregion
+
+        #region Supermatter
+
+        /// <summary>
+        ///     Standard Effect on PowerMixRatio
+        /// </summary>
+        public const float OxygenPowerMixRatio = 1f;
+
+        /// <summary>
+        ///     Standard Effect on PowerMixRatio
+        /// </summary>
+        public const float WaterPowerMixRatio = 1f;
+
+        /// <summary>
+        ///     Standard Effect on PowerMixRatio
+        /// </summary>
+        public const float PlasmaPowerMixRatio = 1f;
+
+        /// <summary>
+        ///     Standard Effect on PowerMixRatio
+        /// </summary>
+        public const float  CO2PowerMixRatio = 1f;
+
+        /// <summary>
+        ///     Reduces PowerMixRatio
+        /// </summary>
+        public const float  NitrogenPowerMixRatio = -1f;
+
+        /// <summary>
+        ///     Standard Effect on PowerMixRatio
+        /// </summary>
+        public const float  TritiumPowerMixRatio = 1f;
```

Comment:
Try to keep style consistent i.e. 1 space between float and the const name.

Also should these be done as prototypes on specific gasses instead? @Zumorica would know more